app/assets/stylesheets/welcome.scss

```diff
@@ -136,3 +132,20 @@ a {
 .octo {
   margin-top: 0px;
 }
+
+#search-dropdown {
+  margin-top: 85px;
+}
+
+.list-group-item {
+  background: #38d5fb;
+  a {
+    color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.